src/normalize.js

```diff
@@ -1,8 +1,6 @@
-import camelCase from 'lodash/camelCase';
-import isArray from 'lodash/isArray';
-import cloneDeep from 'lodash/cloneDeep';
-import keys from 'lodash/keys';
-import merge from 'lodash/merge';
+import {
+  camelCase, isArray, cloneDeep, keys, merge, difference,
```

Comment:
Is that import structure better for tree shaking? Or why the change?

Reply:
I thought they are equivalent, but seem they aren't:
https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/#:~:text=The%20lodash%2Des%20library%20supports,up%20in%20the%20app%20bundle.

Will revert.